### DISCUSSION TITLE
It’ there are some methods to view package source code like jupyter notebook？

### DISCUSSION THREAD

**Discussion by luguoyixiazi** [marimo-team/marimo#747] [Q&A]
I can’t use vscode extension because it can’t find the file, so I have to use marimo with browser, and it's too tied to open a jupyternotebook or a python file in editor for reading source code

**Comment by akshayka** [comment]
>  and it's too tied to open a jupyternotebook or a python file in editor for reading source code

What do you mean by this? Do you mean you want to use the marimo browser editor to view files that are not marimo notebooks, but just regular Python programs?
